dataretrieval/nwis.py

```diff
@@ -35,7 +38,7 @@ def format_response(df, service=None):
         # XXX: consider making site_no index
         return df
 
-    elif len(df['site_no'].unique()) > 1:
+    elif len(df['site_no'].unique()) > 1 and multi_index is None:
```

Comment:
What are you doing with multi_index?  It is only being used as a boolean here.  If it is being used as a boolean then it should be a boolean.

Reply:
If False it will turn off the multi-level indexing.